### PR TITLE
update to latest internal alpine image

### PR DESCRIPTION
**Pull request by davejrt**
follow up from https://github.com/sourcegraph/sourcegraph/pull/27510

### CODE REVIEW

File: cmd/frontend/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # file, please don't be scared to make it more pleasant / remove hadolint
 # ignores.
 
-FROM sourcegraph/alpine-3.12:114329_2021-10-28_bc663f1@sha256:f5ed5ba991b86c8307d8b2cf485585e3246194092b2fd29829bed1b218c23f64
+FROM sourcegraph/alpine-3.12:16223_2021-11-12_dfe1246sha256:e65a7eda595b35f45bc3c2d3abcdfca623b3f4a5b9d7af9a52c3eca6cf0ff3b4
```
Comment (caugustus-sourcegraph) [inline cmd/frontend/Dockerfile:6]:
```suggestion
FROM sourcegraph/alpine-3.12:16223_2021-11-12_dfe1246@sha256:e65a7eda595b35f45bc3c2d3abcdfca623b3f4a5b9d7af9a52c3eca6cf0ff3b4
```
(etc etc)

Comment (davejrt) [reply cmd/frontend/Dockerfile]: big regex fail

**Review by caugustus-sourcegraph — APPROVED** [review]
Not sure if Changelog is required

**Comment by davejrt** [comment]
> Not sure if Changelog is required

Historically speaking, unless we are patching a previous release and this is the reason we haven't bothered. We'd consider this just part of our regular maintenance of images.

**Pull request merged**